server/src/main/java/org/opensearch/search/aggregations/bucket/histogram/InternalHistogram.java

```diff
@@ -382,7 +386,58 @@ private double round(double key) {
         return Math.floor((key - emptyBucketInfo.offset) / emptyBucketInfo.interval) * emptyBucketInfo.interval + emptyBucketInfo.offset;
     }
 
-    private void addEmptyBuckets(List<Bucket> list, ReduceContext reduceContext) {
+    private int estimateTotalBucketCount(List<Bucket> list) {
+        if (emptyBucketInfo == null || list.isEmpty()) {
```

Comment:
I suppose this means `emptyBucketInfo` will exist even when user doesn't define extended bound? I didn't check myself, but pls check once.